### PR TITLE
docs: fix phpdoc types

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
See #6310

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: tests/system/CommonSingleServiceTest.php [hunk 1]
```diff
@@ -36,6 +36,9 @@ public function testSingleServiceWithNoParamsSupplied(string $service): void
         $service1 = single_service($service);
         $service2 = single_service($service);
 
+        assert($service1 !== null);
+        assert($service1 !== null);
```
Comment (samsonasik) [inline tests/system/CommonSingleServiceTest.php:40]: should be service1 and service2 ? Above 2 lines are same (service1)

File: tests/system/CommonSingleServiceTest.php [hunk 2]
```diff
@@ -64,6 +67,9 @@ public function testSingleServiceWithAtLeastOneParamSupplied(string $service): v
         $service1 = single_service($service, ...$params);
         $service2 = single_service($service, ...$params);
 
+        assert($service1 !== null);
+        assert($service1 !== null);
```
Comment (samsonasik) [inline tests/system/CommonSingleServiceTest.php:71]: here too, service1 and 2

File: tests/system/CommonSingleServiceTest.php (hunk 1 above)
Comment (kenjis) [reply tests/system/CommonSingleServiceTest.php]: Yes. it is copy and paste mistake.
But the `$service2` is not needed. So I removed it.

File: tests/system/CommonSingleServiceTest.php (hunk 2 above)
Comment (kenjis) [reply tests/system/CommonSingleServiceTest.php]: Fixed.

**Review by michalsn — APPROVED** [review]

**Pull request merged**